Modules/MainFrame/Frames.lua

```diff
@@ -598,6 +598,85 @@ function Arenalogs.MAIN_FRAME.FRAMES:InitMatchHistoryTab()
             scale = dropdownScale,
         }
     }
+
+    matchHistoryTab.content.characterDropdown = GGUI.CustomDropdown {
+        ---parent = matchHistoryTab.content, anchorParent = matchHistoryTab.content.teamDisplayDropdown.frame.frame.frame,
+        parent = matchHistoryTab.content, anchorParent = matchHistoryTab.content.pvpModeDropdown.frame.frame,
+        anchorA = "LEFT", anchorB = "RIGHT", width = 180, offsetX = 10,
+        initialData = PopulateDropdownChar(),
+        initialLabel = GetCurrentPlayer(),
+        initialValue = GetCurrentPlayer(),
+        clickCallback = function(self, label, value)
+            Arenalogs.MAIN_FRAME.FRAMES:UpdateMatchHistory()
+        end,
+        buttonOptions = {
+            parent = matchHistoryTab.content,
+            buttonTextureOptions = Arenalogs.CONST.ASSETS.BUTTONS.DROPDOWN,
+            fontOptions = {
+                fontFile = Arenalogs.CONST.FONT_FILES.ROBOTO,
+            },
+            sizeY = dropdownSizeY,
+            scale = dropdownScale,
+        },
+        arrowOptions = Arenalogs.CONST.ASSETS.BUTTONS.DROPDOWN_ARROW_OPTIONS,
+        selectionFrameOptions = {
+            backdropOptions = Arenalogs.CONST.DROPDOWN_SELECTION_FRAME_BACKDROP,
+            scale = dropdownScale,
+        }
+    }
+end
+
+function GetPlayerClass(characterName)

```

Comment:
if you define a function like that in lua it will be global
and we dont want that because this could collide with blizzard api or other addons

either use
`local function <name>`
.. to make it accessible in this file only
or
`function SomeTable:<name>`
.. to put it into a table

I usually put functions within a module into the module (which is a table)
so

`function ARENA_DEATH_ANALYZER:GetPlayerClass()`

Although I think a function like this should be put into Arenalogs.UTIL